packages/cdc-react/src/components/Card/Card.tsx

```diff
@@ -0,0 +1,54 @@
+import { ButtonIconPositionTypes, ButtonVariationsTypes } from "src/@types";
+import { Button } from "../Button/Button";
+import "./Card.scss";
+interface CardProps {
+  layout: "horizontal" | "vertical-left" | "vertical-right";
+  action: string;
+  mediaSource?: string;
+  header?: string;
+  sectionTitle?: string;
+  showTitleTop?: boolean;
```

Comment:
Feeling like this prop needs a comment.  Is there a situation where we don't want the title on top?

Reply:
It could be renamed to something like "mediaBeforeTitle". One of the desired layouts is that the image/media is above the title. The title could also be above the image/media too.

Reply:
Maybe `setMediaBanner`?  When the media is before the section title, it isn't really part of the section body anymore, and instead a banner of the whole card.